ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.js

```diff
@@ -28,42 +26,43 @@ import {
   AssetType,
   TokenStandard,
 } from '../../../shared/constants/transaction';
+import { getSuggestedTokens } from '../../selectors';
 
 function getTokenName(name, symbol) {
   return name === undefined ? symbol : `${name} (${symbol})`;
 }
 
 /**
- * @param {Array} suggestedAssets - an array of assets suggested to add to the user's wallet
+ * @param {Array} suggestedTokens - an array of assets suggested to add to the user's wallet
```

Comment:
Is the core of the changes in this file simply to draw the difference between a token asset and other types of assets?